packages/http-server/src/serialize.ts

```diff
@@ -42,6 +42,8 @@ export const serialize = (payload: unknown, contentType?: string) => {
   const serializer = contentType ? serializers.find(s => s.regex.test(contentType)) : undefined;
 
   if (!serializer) {
+    if (typeof payload === 'string') return payload;
```

Comment:
Missing test

Reply:
There's an harness test verifying that in such case the payload is just let go.